qemu.go

```diff
@@ -106,6 +108,11 @@ const (
 	maxDevIDSize = 31
 )
 
+const (
+	// NVDIMM device needs memory space (1024MB is suggested by @grahamwhaley)
```

Comment:
I agree with @jodh-intel, you should not mention @grahamwhaley directly into the source code. And please refer only the isssue, not the very specific comment.